scanners/boostsecurityio/boost-sca/rules.yaml

```diff
@@ -1,5 +1,6 @@
 import:
   - boostsecurityio/sca-cve
+  - boostsecurityio/oss-license
 
 rules:
   dependency-with-malicious-behaviour:
```

Comment:
Wondering if this should be only 1 place as well

Reply:
Probably, can be cleaned up in a later PR.

Reply:
https://github.com/boostsecurityio/dev-registry/pull/133